IBF-Typhoon-model/mainpipeline.py

```diff
@@ -31,7 +31,7 @@
         logging.getLogger(log_name).setLevel(max(logging.WARNING, level))
```

Comment:
This file should be deleted as well (and so should main.py -- there should only be one "main" file, which is currently the one in `scr`)